### PR TITLE
Add a bit of Intel to the POI spawning reward

### LINKED ISSUE #374: Intel rewards are too sparse
Intel is far too scarce, a small amount should be added to the reward that spawns POIs

### PR DESCRIPTION
**Pull request by cannonfodder1**
Closes #374

### CODE REVIEW

File: CovertInfiltration/Localization/XComGame.int [hunk 1]
```diff
@@ -281,8 +281,8 @@ FacilityName="Living Quarters"
 Summary="Increases Barracks capacity by a further 15, allowing more soldiers to be maintained without an increase to recovery times."
 
 [Reward_Rumor X2RewardTemplate]
-DisplayName="Resistance Rumor"
-RewardDetails="Reveal a Point of Interest on the Geoscape"
+DisplayName="Intel and Rumor"
+RewardDetails="Gain Intel and reveal a Point of Interest on the Geoscape"
```
Comment (Xymanek) [inline CovertInfiltration/Localization/XComGame.int]: I think the loc should mention that it's "a bit of intel" or something

Comment (cannonfodder1) [reply CovertInfiltration/Localization/XComGame.int]: I don't actually think the Reward Details string ever shows up anywhere.

Comment (Xymanek) [reply CovertInfiltration/Localization/XComGame.int]: Well, then `DisplayName` should somehow indicate it

Comment (cannonfodder1) [reply CovertInfiltration/Localization/XComGame.int]: It does, the display name is combined with the intel amount on the mission UI. So it reads as `X Intel and Rumor`

Comment (Xymanek) [reply CovertInfiltration/Localization/XComGame.int]: Oh ok

Comment (Xymanek) [reply CovertInfiltration/Localization/XComGame.int]: Something to consider: the UI supports multiple rewards. What is the intel and rumour were separated into individual rewards?

**Pull request merged**